gov_uk_dashboards/components/plotly/table.py

```diff
@@ -22,12 +22,11 @@ def table_from_dataframe(
     Args:
         dataframe (DataFrame): Dataframe containing formatted data to display.
         title (str, optional): Title to display above the table. Defaults to None.
-        include_headers (bool, optional): If the column labels should be included as headers.
```

Comment:
Delayed, but potentially useful feedback.  Removing an argument from a function is a [breaking change](https://stackoverflow.com/a/21703427).

Our [update guidance](https://github.com/communitiesuk/pkg_gov_uk_dashboards#updating) states that we should update the major version if making a breaking change.

Reply:
Thanks for the comment Adrian, that was what I was least sure of!